### PR TITLE
Add CircleCI configuration

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build_and_test:
+    machine: true
+
+    steps:
+    - checkout
+
+    - run:
+        name: Install dep
```
Comment (metacosm) [inline .circleci/config.yml]: Not sure this is needed, actually. Same goes for running `dep ensure`. This should only be needed when developing and changing go imports but checked in versions should be checked with proper, up-to-date vendor / dep files…

Comment (geoand) [reply .circleci/config.yml]: Understood. I will remove this step then

Comment (geoand) [reply .circleci/config.yml]: Fixed

**Comment by metacosm** [comment]
Thank you!

**Comment by geoand** [comment]
No problem :)

**Pull request merged**